data/korath/korath outfits.txt

```diff
@@ -105,7 +132,7 @@ outfit "Systems Core (Small)"
 	"hull repair rate" 0.2
 	"hull energy" 0.2
 	"hull heat" 0.3
-	description `The heart of any Korath ship is the Systems Core, which houses the ship's central computer, shield generators, and a small number of "hull-walking" welding robots that can make minor repairs to the ship's outer armor while in flight.`
+	description `The heart of any Korath ship is the Systems Core, which houses the ship's central computer, shield generators, and a small microbot swarm that can make minor repairs to the ship while in flight.`
```

Comment:
Adding formally to this review: Since there's no indication that the swarm can expire or wear out I would suggest that it actually produces the swarm through some sort of 3D printing technology (something probably similar to stereolithography or DMLS):
```suggestion
	description `The heart of any Korath ship is the Systems Core, which houses the ship's central computer, shield generators, reserve power, and a small reservoir which stores, produces and recycles a small microbot swarmin that can make minor repairs while in flight.`
```